eng/packaging.targets

```
@@ -9,14 +9,34 @@
 
   <!-- .NETCoreApp TFMs < NetMinimum aren't supported and intentionally dropped. -->
   <ItemGroup>
-    <PackageValidationBaselineFrameworkToIgnore Include="netcoreapp2.1;" />
+    <PackageValidationBaselineFrameworkToIgnore Include="net11" />
+    <PackageValidationBaselineFrameworkToIgnore Include="net20" />
+    <PackageValidationBaselineFrameworkToIgnore Include="net35" />
+    <PackageValidationBaselineFrameworkToIgnore Include="net40" />
+    <PackageValidationBaselineFrameworkToIgnore Include="net403" />
+    <PackageValidationBaselineFrameworkToIgnore Include="net45" />
+    <PackageValidationBaselineFrameworkToIgnore Include="net451" />
+    <PackageValidationBaselineFrameworkToIgnore Include="net452" />
```

Other .NET Framework TFMs than these shouldn't show up. I would even be surprised if net35 or net40 is showing up anywhere.

```suggestion
    <PackageValidationBaselineFrameworkToIgnore Include="net35" />
    <PackageValidationBaselineFrameworkToIgnore Include="net40" />
    <PackageValidationBaselineFrameworkToIgnore Include="net45*" />
```